packages/checkout-ui-extensions-run/package.json

```diff
@@ -21,7 +21,7 @@
       "import": "./index.mjs",
       "require": "./index.js"
     },
-    "./": "./"
+    "./*": "./*"
```

Comment:
Actually, there's no harm in shipping this before the packages are published since i'll be merging with `main` anyhow prior to publishing.

I don't have context on the other packages though in this PR. @lemonmade do you feel comfortable approving this?